olm/operator-source.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: cert-utils-operator
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: raffaelespazzoli
+  displayName: "Cert-utils-operator"
+  publisher: "COP"
```

Comment:
Red Hat Community of Practice (CoP)

Reply:
this is just for manual testing.. anyway I'll fix it.